### DISCUSSION TITLE
Players that go full screen should be on top of "always on top" screens

### DISCUSSION THREAD

**Discussion by tmdarlan** [f1multiviewer/issue-tracker#111] [Feature requests] [answered]
or at least have an option. I use always on top on windows to put the windows on top of the tool bar. but i still want full screen to work.

**Comment by JJWatMyself** [comment, marked as answer]
I initially agreed but thinking it through, ff you want full screen plus a PIP window (on-top) I think it is better to leave it the way it is.  If this was changed, then the folks that are looking for PIP wouldn't have a method.  So my suggestion is not to change this behavior.
